pulp_ansible/app/tasks/collections.py

```diff
@@ -421,7 +421,7 @@ def _get_backend_storage_url(artifact_file):
     return url
 
 
-def _update_highest_version(collection_version):
+def _update_highest_version(collection_version, save=False):
```

Comment:
I fail to see the use for `save=False`. Don't we always want to update when we go to the lengths of running this function at all?

Reply:
I addressed this in a previous review comment. @gerrod3 wants to retain the behavior where the function caller does the saving when possible to reduce duplicate saving.

Reply:
Thank you, so we are actually preventing a superfluous save call.